src/components/Section/Relationships/RelationshipStatus/Marital.test.jsx

```diff
@@ -62,4 +63,197 @@ describe('The relationship status component', () => {
+            },
+        valid: false
+      }
+
+      const component =  mount(<Marital {...props} />)
+      expect(component.find('.error-messages [data-i18n="error.civilUnion.min"]').text()).toEqual(
+        `${i18n.t('error.civilUnion.min.title')}${i18n.t('error.civilUnion.min.message')}`
+      )
+    })
+  })
+
+  describe('handles divorce dates', () => {
+    it('with good data', () => {
+      const props = {
+        name: "marital",
+        Status: {
+            checked: true,
+            value: 'Divorced'
+          },
+        applicantBirthdate: {
+          estimated: false,
+          day: "1",
+          month: "1",
+          name: "birthdate",
+          year: "1970",
+          date: new Date("1970", "1", "1")
+        },
+        DivorcedList: {
+          items: [{
+            Item: {
+              Birthdate: {
+                estimated: false,
+                day: "1",
+                month: "1",
+                name: "birthdate",
+                year: "1980",
+                date: new Date("1980", "1", "1")
+              },
+              Recognized: {
+                estimated: false,
+                day: "1",
+                month: "1",
+                name: "Recognized",
+                year: "1990",
+                date: new Date("1990", "1", "1")
+              },
+              DateDivorced: {
+                estimated: false,
+                day: "1",
+                month: "1",
+                name: "DateDivorced",
+                year: "2000",
+                date: new Date("2000", "1", "1")
+              },
+            },
+          open: true
+        }],
+      },
+        valid: true
+      }
+
+      const component =  mount(<Marital {...props} />)
+      expect(component.find('.error-messages [data-i18n="error.divorceDate.min"]').children().length).toEqual(0)
+    })
+    it('with bad data', () => {
+      const props = {
+        name: "marital",
+        Status: {
+            checked: true,
+            value: 'Divorced'
+          },
+        applicantBirthdate: {
+          estimated: false,
+          day: "1",
+          month: "1",
+          name: "birthdate",
+          year: "1970",
+          date: new Date("1970", "1", "1")
+        },
+        DivorcedList: {
+          items: [{
+            Item: {
+              Birthdate: {
+                estimated: false,
+                day: "1",
+                month: "1",
+                name: "birthdate",
+                year: "1980",
+                date: new Date("1980", "1", "1")
+              },
+              Recognized: {
+                estimated: false,
+                day: "1",
+                month: "1",
+                name: "Recognized",
+                year: "1990",
+                date: new Date("1990", "1", "1")
+              },
+              DateDivorced: {
+                estimated: false,
+                day: "1",
+                month: "1",
+                name: "DateDivorced",
```

Comment:
@line47 Just for my background, how are the `name` props at this level, related to the name of the enclosing object? Here they are an exact match, other times they are birthdate/Birthdate – is it just a pervasive inconsistency?